config.py

```diff
@@ -1,6 +1,8 @@
 import logging
 import os
 
+ACCESS_CONTROLS = {'LabHub:create_issue_cmd': {'allowprivate': False}}
```

Comment:
@LifetimeAlone You already have `ACCESS_CONTROLS` glob in *config.py*, why don't you add your changes to the same?
https://github.com/LifetimeAlone/corobo/blob/dev4/config.py#L60

Reply:
Done!